chrome/background_test.js

```diff
@@ -323,7 +325,7 @@ function testOtpMode() {
     if (request.password == 'pw') {
       localStorage['pwhash'] = JSON.stringify(
           {'email': 'adhintz@google.com',
-           'date': 1});
+            'date': 1});
```

Comment:
This indentation looks odd to me. In the merge I'll change it to 
      localStorage['pwhash'] = JSON.stringify({
        'email': 'adhintz@google.com',
        'date': 1});